zerver/views/registration.py

```diff
@@ -204,10 +204,12 @@ def accounts_register(request):
             org_type = int(form.cleaned_data['realm_org_type'])
             realm = do_create_realm(string_id, realm_name, org_type=org_type)[0]
 
+            # Eventually, picking the names, descriptions, and welcome messages will be
+            # done in the realm creation UI, and this data will be coming from a form.
```

Comment:
ah, there was a misunderstanding. The proposal to make this configurable was a proposal to make it configurable for the server admin (not the realm admin).
Regardless, I would leave such a comment out; github issues are a better place to store intended work.